usr/lib/linuxmuster-webui/plugins/lmn_linbo_sync/resources/js/controllers/index.controller.coffee

```diff
@@ -8,8 +8,6 @@ angular.module('lmn.linbo_sync').controller 'SyncIndexController', ($scope, $htt
             $scope.linbo_command[group] = {'cmd':[], 'show':false, 'host': [], 'target':'clients' }
 
     $scope.isUp = (group, host) ->
-        index = $scope.groups[group].hosts.indexOf(host)
```

Comment:
Removing this line will break the `isUp()` function. In order to deal with hostnames prefixed with school, it may be necessary to update the function [handle_api_workstations_online](https://github.com/linuxmuster/linuxmuster-webui7/blob/d45811804ada42c9f17ef4b9ef1b6849756085f4/usr/lib/linuxmuster-webui/plugins/lmn_linbo_sync/views.py#L50)

Reply:
You are right. Seems I missed this somehow in my testing and forgot to compile the javascript code.

Anyhow, adding this e96d05341479209daf245363259bd4948c11c262 and this 83aafc97d59af7f4539da839b684531d971c352f will generate a fitting list.

The handle_api_workstations_online function uses the hostname from the devices list. Computer added in multischool are added from this list as well, just with the added prefix. So this is what I did in the the api.py here e96d05341479209daf245363259bd4948c11c262 and works.

Maybe we should improve this in the future and getting the devices from ad, so we wouldn't have to generate the hostnames. But for now this should work.

I'll update my code.

Reply:
Seems the compiled javascript code still had this line 11.